homeassistant/components/unifi/sensor.py

```diff
@@ -171,6 +177,35 @@ def async_device_state_value_fn(hub: UnifiHub, device: Device) -> str:
     return DEVICE_STATES[device.state]
 
 
+@callback
+def async_client_support_wan_monitor(hub: UnifiHub, obj_id: str) -> bool:
+    """Determine if a client supports WAN monitoring."""
+    device = hub.api.devices[obj_id]
+    if device is None:
+        return False
+
+    uptime_stats = device.raw.get("uptime_stats")
+    if uptime_stats is None:
+        return False
+
+    wan = uptime_stats is not None and uptime_stats.get("WAN")
```

Comment:
I might need some help here @Kane610. 

So I know the `uptime_stats` is not None and it's a `dict`. But how do I tell mypy that?

Reply:
I think I got it, but not sure if it's the right way?

Reply:
I don't think it. 
Can it be because that `TypedDevice` is missing `uptime_stats`?

Reply:
We should add a proper uptime stat to the library so typing is correct